SparkLend Ethereum - Increase Capacity of weETH - Governance Poll 2024-07-01 (#2680)
makerdao/community: governance/polls/SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024.md

---
title: SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024

> summary: Increase capacity of WeETH (Etherfi weETH) on SparkLend Ethereum.

Reviewer: ```suggestion
summary: Increase capacity of weETH (Etherfi weETH) on SparkLend Ethereum.
```

discussion_link: https://forum.makerdao.com/t/jun-27-2024-proposed-changes-to-spark-for-upcoming-spell/24552
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-07-01T16:00:00
end_date: 2024-07-04T16:00:00
---
# Poll: SparkLend Ethereum - Increase Capacity of weETH - July 1, 2024

The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 1 at 16:00 UTC.

**This is a binary vote.**

- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

> This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of weETH (Etherfi weETH) on mainnet with the following parameters:

Reviewer 2: ```suggestion
This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of [weETH (Etherfi weETH)](https://app.spark.fi/market-details/1/0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee) on mainnet with the following parameters:
```

> This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of [weETH (Etherfi weETH)](https://app.spark.fi/market-details/1/0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee) on mainnet with the following parameters:

Reviewer 2: ```suggestion
This poll allows MakerDAO Governance to signal their support or opposition to increasing the capacity of [weETH (Etherfi weETH)](https://app.spark.fi/market-details/1/0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee) on mainnet with the following parameter changes:
```

> - weETH Supply Cap Max: **200,000 weETH**

Reviewer 2: ```suggestion
- Increase weETH Supply Cap Max by 150,000 weETH, from 50,000 weETH to **200,000 weETH**
```

> - weETH Isolation Mode Debt Ceiling: **200 million DAI**

Reviewer 2: ```suggestion
- Increase weETH Isolation Mode Debt Ceiling by  150 million DAI, from 50 million DAI to **200 million DAI**
```

> - Increase weETH Isolation Mode Debt Ceiling by 150 million DAI, from 50 million DAI to **200 million DAI**

Reviewer 2: ```suggestion
- Increase weETH [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features) Debt Ceiling by 150 million DAI, from 50 million DAI to **200 million DAI**
```
